process/interceptors/processor/txInterceptorProcessor.go

```diff
@@ -0,0 +1,47 @@
+package processor
+
+import (
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+// TxInterceptorProcessor is the interceptor used for transaction-like structs that satisfy
+type TxInterceptorProcessor struct {
```

Comment:
TxInterceptor ? delete transaction-like. Used for intercepting transactioins (smart contract results, receipts, transaction) structs which satisfy TransactionHandler interface.

Reply:
updated comment. I would leave TxInterceptorProcessor because it satisfies InterceptorProcessor interface